editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1535,10 +1533,16 @@ export const MetadataUtils = {
       openFile,
     )
 
-    return {
+    workingElements = {
       ...workingElements,
       ...spyOnlyElements,
     }
+
+    const elementsInheritingFromAncestors = fillMissingDataFromAncestors(workingElements) // fontos hogy a workingElements-et adjuk neki
```

Comment:
Errr...

Reply:
Blame Balazs!

Reply:
no no no, this is your PR Berci :D the blame deflects from me :D